src/main/java/pl/itacademy/schedule/ScheduleGeneratorApp.java

```diff
@@ -1,29 +1,70 @@
 package pl.itacademy.schedule;
 
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.time.format.DateTimeFormatter;
+
 import org.apache.commons.cli.ParseException;
+import org.apache.poi.ss.usermodel.Workbook;
+
 import pl.itacademy.schedule.exception.IncorrectParametersException;
+import pl.itacademy.schedule.generator.ExcelCreator;
+import pl.itacademy.schedule.generator.Schedule;
+import pl.itacademy.schedule.generator.ScheduleGenerator;
 import pl.itacademy.schedule.parameters.EnteredParameters;
 import pl.itacademy.schedule.parameters.ParametersReader;
+import pl.itacademy.schedule.util.PropertiesReader;
 
 public class ScheduleGeneratorApp {
 
-    public static void main(String[] args) {
-
-        ParametersReader.UsagePrinter usagePrinter = new ParametersReader.UsagePrinter();
-        ParametersReader parametersReader = new ParametersReader();
-        EnteredParameters enteredParameters = null;
-        try {
-            enteredParameters = parametersReader.parseArguments(args);
-            if (enteredParameters.isShowHelp()) {
-                usagePrinter.printHelp();
-                return;
-            }
-        } catch (IncorrectParametersException | ParseException  e) {
-            System.out.println(e.getMessage());
-            usagePrinter.printHelp();
-        } catch (NumberFormatException e) {
-            System.out.println("Impossible to read number " + e.getMessage());
-        }
-
-    }
+	public static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("d.MM.uuuu");
+	public static final DateTimeFormatter TIME_FORMATTER = DateTimeFormatter.ofPattern("H:mm");
+
+	public static void main(String[] args) {
+
+		ParametersReader.UsagePrinter usagePrinter = new ParametersReader.UsagePrinter();
+		ParametersReader parametersReader = new ParametersReader();
+		EnteredParameters enteredParameters = null;
+		try {
```

Comment:
We don't want to keep all the stuff inside of single try section.
If, for example, we will not be able to write Excel file because of file-system problem, application usage (from catch section) will not help us.

Reply:
WIll do it when all parts are ready.